apps/account-worker/src/transaction_notifier/notifier.service.ts

```diff
@@ -22,7 +22,7 @@ import { PinoLogger } from 'nestjs-pino';
 
 @Injectable()
 export class TxnNotifierService
-  extends BlockchainScannerService
+  extends AccountBlockchainScannerService
```

Comment:
Renamed for clarity - this one is unique and uses `BlockchainService`, and _that_ inherits from libs/blockchain/blockchain-scanner.service.  `BlockchainService` does extra things unique to `AccountBlockchainScannerService`.

Reply:
note, `BlockchainService` does not inherit `BlockchainScannerService`